Model/Sync/Reset.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Yotpo\Core\Model\Sync;
+
+use Yotpo\Core\Model\Sync\Reset\Catalog;
+use Yotpo\Core\Model\Sync\Reset\Customers;
+use Yotpo\Core\Model\Sync\Reset\Orders;
+
+class Reset
```

Comment:
The class name is very implicit. Reset what exacly?
What about `ResetEntitiesSync`?

Reply:
This is understood. We will make the changes and update the PR soon